### ISSUE TITLE
Can't parse select a,b,c from table group by rollup((a,b,c))

### ISSUE THREAD

**Issue by anycs** [JSQLParser/JSqlParser#1210]
I have select SQL for example：select a,b,c from table group by rollup((a,b,c))，if use rollup((a,b,c)) 

I will get error :Caused by: net.sf.jsqlparser.parser.ParseException: Encountered " "(" "( ""
but this SQL can be executed normally in Oracle 

I'm very anxious about that. Thank you

**Comment by wumpz** [comment]
I assume the problem are those double parenthesis. Does it work for 

    select a,b,c from table group by rollup(a,b,c)

**Comment by manticore-projects** [comment]
Currently we are parsing a SimpleExpression and we should change that into a ComplexExpression:
```
{
    <K_GROUP> <K_BY>
    (
        columnReference=SimpleExpression() {groupBy.addGroupByExpression(columnReference); }
        ("," columnReference=SimpleExpression()  {groupBy.addGroupByExpression(columnReference); } )*
    |
    ...  
```

**Comment by manticore-projects** [comment]
Solved by PR #1218

**Comment by anycs** [comment]
Thank you
